engine/src/main/resources/application.conf

```diff
@@ -78,86 +70,128 @@ call-caching {
-  // In that scheme, a client_id and client_secrets must be provided. They must be the ones that have been used to generate the above-mentioned refresh token.
-  // userAuthenticationScheme = "refresh"
-  // refreshTokenAuth = {
-  //   client_id = ""
-  //   client_secret = ""
-  // }
+backend {
+  default = "Local"
+  providers = [
+    {
+      name = "Local"
+      class = "cromwell.engine.backend.local.LocalBackend"
+      config {
+        // Root directory where Cromwell writes job results.  This directory must be
+        // visible and writeable by the Cromwell process as well as the jobs that Cromwell
+        // launches.
+        root: "cromwell-executions"
+
+        filesystems = {
+          local {
+            // Cromwell makes a link to your input files within <root>/<workflow UUID>/workflow-inputs
+            // The following are strategies used to make those links.  They are ordered.  If one fails
+            // The next one is tried:
+            //
+            // hard-link: attempt to create a hard-link to the file
+            // copy: copy the file
+            // soft-link: create a symbolic link to the file
+            //
+            // NOTE: soft-link will be skipped for Docker jobs
+            localization: [
+              "hard-link", "soft-link", "copy"
+            ]
+          }
+          gcs {
+            // References an auth scheme defined in the `google` stanza.
+            auth = "application-default"
+          }
+        }
+      }
+    },
+    {
+      name = "SGE"
+      class = "cromwell.engine.backend.sge.SgeBackend"
+      config {
+        // Root directory where Cromwell writes job results.  This directory must be
+        // visible and writeable by the Cromwell process as well as the jobs that Cromwell
+        // launches.
+        root: "cromwell-executions"
+
+        // Cromwell makes a link to your input files within <root>/<workflow UUID>/workflow-inputs
+        // The following are strategies used to make those links.  They are ordered.  If one fails
+        // The next one is tried:
+        //
+        // hard-link: attempt to create a hard-link to the file
+        // copy: copy the file
+        // soft-link: create a symbolic link to the file
+        //
+        // NOTE: soft-link will be skipped for Docker jobs
+        filesystems = {
+          local {
+            localization: [
+              "hard-link", "soft-link", "copy"
+            ]
+          }
+        }
+      }
+    },
+    {
+      name = "JES"
+      class = "cromwell.engine.backend.jes.JesBackend"
+      config {
+        // Google project
+        project = "my-cromwell-workflows"
+
+        // Base bucket for workflow executions
+        root = "gs://my-cromwell-workflows-bucket"
+
+        // Endpoint for APIs, no reason to change this unless directed by Google.
+        endpoint-url = "https://genomics.googleapis.com/"
```

Comment:
Having it here allows for different JES backend instances which can point to production or staging within the same Cromwell.